sdcm/utils/gce_utils.py

```diff
@@ -28,13 +28,21 @@
 
 GceDriver = get_driver(Provider.GCE)  # pylint: disable=invalid-name
 
+# The keys are the region name, the value is the available zones, which will be used for random.choice()
+SUPPORT_REGIONS = {
+    # us-east1 zones: b, c, and d. Details: https://cloud.google.com/compute/docs/regions-zones#locations
+    # choose zones c and d twice as often as zone b
+    'us-east1': 'bccdd',
+    'us-west1': 'abc'}
+
 
 def append_zone(region: str) -> str:
-    assert region.startswith("us-east1"), "only `us-east1' region is supported"
-    if region.count("-") == 1:
-        # us-east1 zones: b, c, and d. Details: https://cloud.google.com/compute/docs/regions-zones#locations
-        return f"{region}-{random.choice('bccdd')}"  # choose zones c and d twice as often as zone b
-    return region
+    for support_region, zone_string in SUPPORT_REGIONS.items():
+        if region == support_region:
+            if region.count("-") != 1:
+                return region
+            return f"{region}-{random.choice(zone_string)}"
+    raise Exception(f'Unsupported region: {region}')
```

Comment:
If exception is raised, then we need another PR to scylla-pkg, which will switch the jobs which runs on gce to use only those 2 datacenters by default. Otherwise, the job continue to failed due to wrong region.

Reply:
Releng will handle this, they have the same problem for AWS and they solved it by adding a list of supported regions for SCT tests